lib/modules/manager/gradle/extract.ts

```diff
@@ -100,18 +100,33 @@ export async function extractAllPackageFiles(
     const key = dep.managerData?.packageFile;
     // istanbul ignore else
     if (key) {
-      const pkgFile: PackageFile = packageFilesByName[key];
-      const { deps } = pkgFile;
-      deps.push({
-        ...dep,
-        registryUrls: [
-          ...new Set([
-            ...defaultRegistryUrls,
-            ...(dep.registryUrls ?? []),
-            ...registryUrls,
-          ]),
-        ],
-      });
+      let pkgFile: PackageFile = packageFilesByName[key];
```

Comment:
```suggestion
      let pkgFile = packageFilesByName[key];
```
i think we don't need this explicit type here